chicagoland.pandemicresponsecommons.org/manifest.json

```diff
@@ -7,7 +7,7 @@
     "arborist": "quay.io/cdis/arborist:2020.03",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
     "covid19-etl": "quay.io/cdis/covid19-etl:1.0.9",
```

Comment:
```suggestion
    "covid19-etl": "quay.io/cdis/covid19-etl:1.0.11",
```
To match the versions, it's already was tested in qa.

Reply:
but it's not the latest? QA should use 1.0.11 to get the fixed data